src/styles/blocks/cash.scss

```diff
@@ -0,0 +1,20 @@
+.cash {
+  display: flex;
+  height: 19px;
+  justify-content: space-between;
+  align-items: center;
+  padding-inline: $standart-inline-pad;
```

Comment:
The variable name '$standart-inline-pad' contains a typo. It should be '$standard-inline-pad' to ensure correct spelling and consistency across your styles.